django_fsm_log/admin.py

```diff
@@ -30,6 +30,6 @@ def get_readonly_fields(self, request, obj=None):
         return self.fields
 
     def get_queryset(self, request):
-        return super(StateLogInline, self).get_queryset(
+        return super().get_queryset(
             request
-        ).order_by(F('timestamp').desc())
```

Comment:
I'd rather keep it that way.

Reply:
Agree with @ticosax . @knaperek might you consider adjust your PR which otherwise is fully legit?

Reply:
Alright, done and rebased.